### DISCUSSION TITLE
Add local datafiles to 3d model when running locally

### DISCUSSION THREAD

**Discussion by jklau1** [StrandedKitty/streets-gl#118] [General]
Hi, 

This project is really impressive. I am blown away. I was wondering if you could help me add our own layers on top of the 3d layer. I have pulled the repository and I am running the project locally on my own server but have not been able to figure out how to add a new layer to the 3d model. What I have in mind is to use the 3d model as a base layer and display other data on top of the layer like crime data (points). Please let me know if I should elaborate further. Looking forwards to hearing back!

**Comment by StrandedKitty** [comment]
Hi. This project is not a library, there are no plans to add such functionality that allows to display custom data. Moreover, the rendering library (https://github.com/StrandedKitty/streets-gl/tree/dev/src/lib/renderer) is quite low-level and lacks handy abstractions to render arbitrary 3D models, points, etc. So adding support for custom data and providing a usable API for it would require a lot of effort.
